src/components/Common/MarkdownDescription.tsx

```diff
@@ -33,7 +33,7 @@ export const EntityMarkdownDescription = ({
 
   return (
     <MemoizedReactMarkdown
-      className="prose-sm text-xs prose-a:text-blue-500 hover:prose-a:underline"
+      className="prose-sm text-xs hover:prose-a:underline"
```

Comment:
it looks like `text-accent-primary` is missing here

Reply:
I set this directly in tailwind config, so we don't need it here